unlock-app/src/components/interface/checkout/alpha/Checkout/useCheckoutItems.ts

```diff
@@ -33,13 +33,20 @@ export function useCheckoutSteps(service: CheckoutService) {
       skip: !paywallConfig.messageToSign,
       to: 'MESSAGE_TO_SIGN',
     },
-    {
-      id: 6,
-      name: 'Solve captcha',
-      to: 'CAPTCHA',
-      skip:
-        !paywallConfig.captcha || ['card', 'claim'].includes(payment.method),
-    },
+    paywallConfig.password
```

Comment:
I think we should move this inside the `lock` object because it's lock specific.